crates/lib3h/src/transport/memory_mock/ghost_transport_memory.rs

```diff
@@ -48,14 +47,14 @@ pub type GhostTransportMemoryEndpointContextParent = GhostContextEndpoint<
 
 #[allow(dead_code)]
 pub struct GhostTransportMemory {
-    machine_id: Address,
+    transport_id: Address,
     network: Arc<Mutex<MemoryNet>>,
     endpoint_parent: Option<GhostTransportMemoryEndpoint>,
     endpoint_self: Detach<GhostTransportMemoryEndpointContext>,
     /// My peer uri on the network layer (not None after a bind)
-    maybe_my_address: Option<Url>,
+    maybe_my_address: Option<Lib3hUri>,
```

Comment:
Is this the same as `Lib3hUrl::undefined("")` now, or not?

Reply:
No it doesn't replace the option as it is about being bound or not.